comixed-rest-api/src/main/java/org/comixedproject/controller/comic/ComicFormatController.java

```diff
@@ -35,20 +39,19 @@
 @RestController
 @Log4j2
 public class ComicFormatController {
-  private static final String LOAD_COMIC_FORMATS = "load.comic-formats";
-  private static final String ADD_COMIC_FORMAT_QUEUE = "/queue/comic-format.add";
 
   @Autowired private ComicFormatService comicFormatService;
   @Autowired private SimpMessagingTemplate messagingTemplate;
 
   /** Retrieves the list of all comic formats and publishes them. */
   @MessageMapping(LOAD_COMIC_FORMATS)
```

Comment:
Need to change the constant with LOAD_COMIC_LIST_MESSAGE

Reply:
@BRUCELLA2 I missed committing some local changes before pushing the PR. They're up now.